practical_05/word_occurrences.py

```diff
@@ -13,7 +13,7 @@
         longest_word = word
 
 for word, count in word_to_count.items():
-    strings.append(("{:{}} : {}".format(word, len(longest_word),word_to_count[word])))
+    strings.append(("{:{}} : {}".format(word, len(longest_word), word_to_count[word])))
```

Comment:
Is creating the list 'strings', and then appending each dictionary pair to that list, and then iterating through the list to print the dictionary pairs the best way? Does it create more memory usage than necessary? Is it possible to print the dictionary pairs taking advantage of the dictionary methods rather copying data and saving as another list and then iterating through that list?